lib/api.v8.js

```diff
@@ -50,9 +50,9 @@ api.v8.OPT_STATUS = [
   /*7*/ 'turbofan optimized'
 ];
 
-api.v8.optimizationStatus = (status) => api.v8.OPT_STATUS[status];
+api.v8.optimizationStatus = status => api.v8.OPT_STATUS[status];
 
-api.v8.getOptimizationToString = (fn) => {
+api.v8.getOptimizationToString = fn => {
```

Comment:
As for me, it's better to always use parentheses when a function body is a block and not expression.